hudi-client/src/main/java/org/apache/hudi/config/HoodieMetricsConfig.java

```diff
@@ -92,28 +103,63 @@ public Builder withReporterType(String reporterType) {
       return this;
     }
 
-    public Builder toGraphiteHost(String host) {
+    public Builder withGraphiteHost(String host) {
```

Comment:
I have two opinions about these changes:

- Does it break the compatibility?
- Packaging different config options of different reporters is not a good choice, it would be better to refactor.

Reply:
These builder methods are not used by any logic. I think they are legacy code and only mentioned in config docs. I'm changing the names in #1603 too. 

You're right that we should segregate the configs for different reporter types. It shouldn't give too much hassle to do refactoring.